README.md

```diff
@@ -167,8 +167,11 @@ The word, gitbook and epub versions might be developed at a later time.
 ## Feedback
```

Comment:
Maybe add a `qData::plot_releases("jhollway/iheiddown")` call to the README? Would need to add this to a new README.Rmd file though and then have a new render step in the pushrelease workflow